src/Components/Attributes/test/Attributes.js

```diff
@@ -34,4 +36,81 @@ describe('Attributes', () => {
       });
     });
   });
+
+  describe('#minimSupport', () => {
+    const minimNamespace = minim.namespace().use(minimParseResult);
+    const Category = minimNamespace.getElementClass('category');
+    const DataStructure = minimNamespace.getElementClass('dataStructure');
+
+    const dataStructures = new Category(
+      new minim.ArrayElement([
+        new DataStructure(
+          new minim.ObjectElement({
+            name: 'Doe',
+            address: 'Address',
+          }, { id: 'User' })
+        ),
+        new DataStructure(
+          new minim.ObjectElement({
+            street: 'Main St.',
+            city: 'Prague',
+            zip: 11150,
+          }, { id: 'Address' })
+        ),
+      ]),
+      { classes: ['dataStructures'] }
+    );
+
+    // How to construct { element: 'Address' } through minim
+    const addressPointer = dataStructures.get(0).content.content[1].value;
+    addressPointer.element = 'Address'; // override
+    delete addressPointer.content;
```

Comment:
@kylef This is a hack to get the following `minim.MemberElement` refract (a reference to another `dataStructure` as observed in `.apib` `fury.parse` output):

```
{
    "element": "member",
    "content": {
        "key": {
            "element": "string",
            "content": "address"
        },
        "value": {
            "element": "Address"
        }
    }
}
```

I couldn't find a better way. Any hints?

Reply:
Yes, I'd rather construct this from the scratch that doing a mutations on refract elements. If I understand correctly we just need to construct the minim tree correctly with reference. Pls find a more proper way how to do that